docs/source/peer-chaincode-devmode.md

```diff
@@ -33,7 +33,15 @@ Throughout this tutorial, all commands are performed from the `fabric/` folder.
     ```
     export FABRIC_CFG_PATH=$(pwd)/sampleconfig
     ```
-5. Generate the genesis block for the ordering service. Run the following command to generate the genesis block and store it in `$(pwd)/sampleconfig/genesisblock` so that it can be used by the orderer in the next step when the orderer is started.
+
+5. Create the `hyperledger` subdirectory in the `/var` directory. This is a location Fabric uses. To create the hyperledger subdirectory, execute these commands, replacing the question marks with your username:
```

Comment:
Can you clarify what Fabric uses in /var/hyperledger. I assume you are talking about the orderer and peer ledger location? Assuming the configtxgen step doesn't need it, I'd move this down to right before starting the orderer and peer.

Reply:
>  I assume you are talking about the orderer and peer ledger location

Exactly